Cloud Pak for Data/WML/notebooks/binary/spark/Watson OpenScale and Watson ML Engine.ipynb

```diff
@@ -92,11 +92,12 @@
    "source": [
     "!pip install --upgrade pyspark==3.0.3 --no-cache | tail -n 1\n",
```

Comment:
We should install pyspark 3.2.x because that's what is given below `spark-mllib_3.2`